content/blog/get-started-with-prometheus-and-grafana-on-docker-with-hpe-storage-array-exporter.md

```diff
@@ -0,0 +1,162 @@
+---
+title: Get started with Prometheus and Grafana on Docker with HPE Storage Array
+  Exporter
+date: 2021-12-07T16:00:00.000Z
+author: Michael Mattsson
+authorimage: /img/portrait-192.jpg
+tags:
+  - HPE Alletra
+  - HPE Nimble Storage
+  - Kubernetes
+  - Prometheus
+---
+With the recently released HPE Storage Array Exporter for Prometheus and HPE CSI Info Metrics Provider for Prometheus, it's a good time to familiarize ourselves with the cloud native technologies involved and get some first hand experience.
```

Comment:
"first hand" should be either "firsthand" or "first-hand"; choose whichever you prefer